e2e/playwright.config.js

```diff
@@ -21,7 +21,11 @@ export default defineConfig({
   // Opt out of parallel tests on CI.
   workers: process.env.CI ? 1 : undefined,
   // Use 'blob' for CI to allow merging of reports. See https://playwright.dev/docs/test-reporters
-  reporter: process.env.CI ? 'blob' : 'html',
+  reporter: process.env.CI
+  ? [['blob']]  :
+    // Never open the HTML report to prevent terminal from hanging on test failure
+    [['html', { open: 'never' }],
+  ],
   // Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions.
   use: {
     // Base URL to use in actions like `await page.goto('/')`.
```

Comment:
The terminal also hangs on `make e2e-show-report` - wonder if we want it to close there too?